### PR TITLE
Feat/delete member

### PR DESCRIPTION
**Pull request by LeftyLincoln**
# Description
-Completes deleteName mutation 

## Type Of Change

- [ ]  Bug fix
- [X]  New feature
- [ ]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [ ]  local host
- [X]  dev tools
- [ ]  cypress

# Related Issues:

- [ ]  closes  

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { useQuery } from "@apollo/client";
 
 const Dashboard = ({ email, setId }) => {
   const { loading, error, data } = useQuery(GET_HOUSEHOLD, {
+    fetchPolicy: "no-cache",
```
Comment (qrispi) [inline src/Components/Dashboard/Dashboard.js:10]: Tricky apollo....

**Review by qrispi — APPROVED** [review]
LGTM! Good work!

File: src/Components/HouseForm/HouseForm.js [hunk 2]
```diff
@@ -12,21 +13,24 @@ const HouseForm = ({ id, email }) => {
   const [editMember, setEditMember] = useState(false);
 
   const { loading: queryLoading, data: queryData, error: queryError } = useQuery(GET_HOUSE_INFO, {
-    variables: { email },
+      fetchPolicy: "no-cache",
```
Comment (Kanderson58) [inline src/Components/HouseForm/HouseForm.js:16]: Big brain moment from Chris looking into the memory cache

**Review by Kanderson58 — COMMENTED** [review]
LGTM!

**Pull request merged**